tools/setup_xcode_sdks.py

```diff
@@ -0,0 +1,200 @@
+#!/bin/python
+
+# Update these lists if you need different SDK versions!
+
+iphoneos_versions = ["10.0", "9.3", "9.2", "8.2"]
```

Comment:
@peter-b I'm wondering if this can be implemented in a way that just looks for SDKs rather than requiring maintenance when there's a new SDK?